.github/workflows/sycl_precommit.yml

```diff
@@ -9,6 +9,8 @@ on:
     - '.github/ISSUE_TEMPLATE/**'
     - '.github/CODEOWNERS'
     - '.github/workflows/sycl_update_gpu_driver.yml'
+    - '.github/workflows/sycl_windows_build_and_test.yml'
+    - '.github/workflows/sycl_macos_build_and_test.yml'
```

Comment:
This might hit us. For instance, if we merge https://github.com/intel/llvm/pull/6862, there might be missed regressions during pre-commit.

Reply:
Right. It's matching the current status. If #6862 lands windows should be removed here. I don't think we plan to land #6862 though... as agreed we should improve post-commit notifications first. Maybe I should close it for now.